metaflow/plugins/metadata/service.py

```diff
@@ -260,7 +260,8 @@ def _version(cls, monitor):
         if cls.INFO is None:
             raise MetaflowException('Missing Metaflow Service URL. '
                 'Specify with METAFLOW_SERVICE_URL environment variable')
-        url = os.path.join(cls.INFO, 'ping')
+        path = 'ping/'
+        url = os.path.join(cls.INFO, path.lstrip('/'))
```

Comment:
lstrip is superfluous in this case. We can just set path = 'ping' and remove all lstrip.

Reply:
The reason I added `/` is to make sure that the error message also has a `/` suffix similar to the `_request` method above. I don't feel strongly about it, so if you don't prefer `/`, I can yank it out.

Reply:
ah, ok, but then maybe rstrip :) (the lstrip is absolutely not doing anything).

Reply:
oh shoot. that was a brainfreeze.